pyproject.toml

```diff
@@ -29,7 +29,7 @@ packages = [{ include = "rstcloth" },]
 python = ">=3.7,<3.12"
 sphinx = ">=2,<6"
 Pygments = "^2.12.0"
-PyYAML = "^6.0"
+PyYAML = "^5.0,^6.0"
```

Comment:
I didn't know you could do this form, does it install ^6.0 by default if there are no other constraints?

- If so, let's keep it, as it's more expressive than ">=5,<7". 
- If not, please can we use ">=5,<7" consistent with the other dependencies?

Reply:
Seems like it didn't work to well in CI. Switched to the normal syntax.